libraries/commerce/product/selectors/product.spec.js

```diff
@@ -53,7 +50,7 @@ describe('Product selectors', () => {
         properties: [property1.label],
       };
 
-      const properties = getProductProperties(mockedState);
+      const properties = getProductProperties(mockedState, { productId: 'product1' });
```

Comment:
Since these mocked props are the same at every place they can be elevated to a variable.

Reply:
You're right. Done.